src/_gettsim/demographics.py

```diff
@@ -157,6 +168,7 @@ def kind_bis_17(alter: int, kind: bool) -> bool:
     return out
 
 
+@policy_function()
 def kind_bis_24(alter: int) -> bool:
```

Comment:
```suggestion
def alter_bis_24(alter: int) -> bool:
```
Now I finally get what has been confusing me... There is no reference to being a child here whatsoever. Contrast to the others defined above.